src/Apps/Sell/Routes/NewRoute/SellArtistAutocomplete.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  AutocompleteInput,
+  AutocompleteInputOptionType,
+  Box,
+  Flex,
+  Image,
+  Text,
+} from "@artsy/palette"
+import { debounce } from "lodash"
+import { useEffect, useMemo, useState } from "react"
+import { Environment, fetchQuery, graphql } from "react-relay"
+import { useSystemContext } from "System/useSystemContext"
+import { extractNodes } from "Utils/extractNodes"
+import {
+  SellArtistAutocomplete_SearchConnection_Query,
+  SellArtistAutocomplete_SearchConnection_Query$data,
+} from "__generated__/SellArtistAutocomplete_SearchConnection_Query.graphql"
+
+const DEBOUNCE_DELAY = 300
+
+export type SellAutocompleteArtist =
+  | NonNullable<
+      NonNullable<
+        NonNullable<
+          NonNullable<
+            SellArtistAutocomplete_SearchConnection_Query$data["searchConnection"]
+          >["edges"]
+        >[number]
+      >["node"]
+    >
+  | null
+  | undefined
+
+interface SellArtistAutocompleteOption extends AutocompleteInputOptionType {
+  option: SellAutocompleteArtist
+}
+
+export const SellArtistAutocomplete: React.FC<{
```

Comment:
As is, that component is not immediately compatible because it's dependent on the Formik, which makes is less portable / reusable. I think ideally we'd create an abstract component which we can wrap for either Formik usage or non-Formik usage.

Although I see some opportunity to reduce the surface size of this component right away by exposing some more elements from the ArtistAutoComplete component.

Reply:
Now using the ArtistAutoComplete component from the Consign app 👍🏼